config/config.go

```diff
@@ -85,7 +80,7 @@ func Build(name string, args []string) (*Config, error) {
 		return nil, errors.New("--tls-secret is required")
 	}
 
-	if conf.Rode.Host == "" {
+	if conf.ClientConfig.Rode.Host == "" {
```

Comment:
Isn't the common client [already performing this check](https://github.com/rode/rode/blob/3ab1e4791cb75459fbf74c71fbeb0062218e9151/common/client.go#L35-L37)?

Reply:
good catch, I'll remove this entirely